packages/@lwc/compiler/src/transformers/__tests__/transform-javascript.spec.ts

```diff
@@ -51,3 +51,139 @@ it('should object spread', async () => {
     expect(code).toContain('b: 1');
     expect(code).not.toContain('...a');
 });
+
+describe('should transform TypeScript file correctly', () => {
```

Comment:
Add a negative test case to show what happens when a TS failure occurs(I understand that TS validation is not part of the oss compiler. But lets establish that oss compiler still works fine even if there is invalid type specification.